cs3api4lab/handlers.py

```diff
@@ -271,8 +271,7 @@ def get(self):
     (r"/api/cs3/public/shares/list", ListPublicSharesHandler),
     (r"/api/cs3/public/share", GetPublicShareByTokenHandler),
     (r"/api/cs3/ocm", OcmSharesHandler),
-    (r"/api/cs3/ocm/received", OcmReceivedSharesHandler)
-    (r"/api/cs3/public/share", GetPublicShareByTokenHandler),
```

Comment:
I don't understand what you're doing here.. Why are you removing the public links endpoint?
What is this PR trying to solve?

Reply:
This commit delete duplicate line: 
`(r"/api/cs3/ocm/received", OcmReceivedSharesHandler)`
and fixes error: 
`
File "\cs3api4lab\__init__.py", line 5, in <module> from .handlers import handlers
File "\cs3api4lab\handlers.py", line 274, in <module> (r"/api/cs3/ocm/received", OcmReceivedSharesHandler)
TypeError: 'tuple' object is not callable
`